packages/client-adapters/src/transformers/alpheios-lexicon-transformer.js

```diff
@@ -177,7 +177,9 @@ class AlpheiosLexiconTransformer {
         const index = entry[0]
         const elem = entry[1]
 
-        const lemmaText = elem.hdwd ? elem.hdwd.$ : undefined
+        // if the parser has a number in the hdwd the JSON will have it as a number
+        // and not a String
+        const lemmaText = elem.hdwd ? `${elem.hdwd.$}` : undefined

```

Comment:
Can we always be sure that the value of `$` is set? Because if `$` is not present in `elem.hdwd` then the value of `${elem.hdwd.$}` will be a string containing "undefined". This is probably not what we want.

So if we're not sure about the presence and value of `$` maybe we can add an extra check? Like `const lemmaText = elem.hdwd && elem.hdwd.$ ? `${elem.hdwd.$}` : undefined`? What do you think?

Reply:
it's a good suggestion. will make this change.